ec3730/Controllers/PingViewController.swift

```diff
@@ -347,12 +350,20 @@ struct PingSwiftUIViewController: View {
                 })
             })
             ToolbarItem(placement: .navigationBarTrailing, content: {
-                NavigationLink(
-                    destination: PingSetList(),
-                    label: {
+                NavigationLink(isActive: $isActive) {
+                    PingSetList()
+                } label: {
+                    Button {
+                        if pings.isEmpty {
```

Comment:
Wouldn't `.disabled` be better than this?

Reply:
Please check, updated it.